litellm/__init__.py

```diff
@@ -925,6 +949,7 @@ def add_known_models():
     "bedrock": bedrock_models + bedrock_converse_models,
     "petals": petals_models,
     "ollama": ollama_models,
+    "ollama_chat": ollama_models,
```

Comment:
This was required for `ollama_chat/*` to work, keeping it symmetrical w/ support for `ollama_chat/MODELNAME`.